findmyparadise/src/main/java/com/tp/wrc/findmyparadise/models/Listing.java

```diff
@@ -15,6 +14,8 @@
     private Double latitude;
     @Column(name = "longitude")
     private Double longitude;
+
```

Comment:
Listing may also need to have a reference to a List of Reviews, Photos, Reservations, etc. I'm just not 100% sure about our table relationships.